mirror_builder/sdist.py

```diff
@@ -149,6 +149,7 @@ def download_sdist(ctx, requirement):
             resolvelib.RequirementsConflicted,
             resolvelib.ResolutionImpossible) as err:
         logger.warning(f'could not resolve {requirement}: {err}')
+        raise err
```

Comment:
Do we want just `raise` here so the rest of the context of the exception is retained? As it is, it will look like the exception originated from this line.

Reply:
Thanks, I'd forgotten about that